### ISSUE TITLE
Need a way to support mulitple include paths

### ISSUE THREAD

**Issue by elwinar** [deuill/go-php#20]
I'm currently working in a docker to compile and run tests, and it happens that the docker container for golang is based upon Debian Jessie.

On Jessie (and probably many other distributions), the correct include path for `main/php.h` is `/usr/include/php5`, which is not the value given in the various CGO headers.

Any though on how to make this more portable ?

**Comment by deuill** [comment]
Hmmmm, indeed. The most portable solution would be adding something like `-I /usr/include/php5` in the [`php5.go file`](https://github.com/deuill/go-php/blob/99962f2bdfab9251e8a57caee772ecda86e9f2e1/engine/php5.go#L5), as incorrect include directives don't throw errors, I believe.

Could you try that locally and respond on whether it works for you or not? I can work on a patch if it does, or look into it further if it doesn't.

**P.S.:** It seems that versions of Debian with PHP7 packages (`testing` and `unstable` for now) place the files in `/usr/include/php` as expected, so there's no fix required for PHP7.

**Comment by elwinar** [comment]
I've already done that locally, and that works.

Something along these lines while in the 
`vendor/github.com/deuill/go-php` directory:

```
sed -i '/s/\/usr\/include\/php/\/usr\/include\/php5/g' **/*.go
```

Thanks for the quick reply.

**Comment by deuill** [comment]
No problem, I'll add that directive and close this once the commit goes in. Happy hacking.

**Comment by deuill** [comment]
(Hopefullly) closed by 0882169cff2bc982773234e7f85c774a8cb015df. Feel free to open a new issue if this doesn't work for you.
